tembo-cli/src/cli/context.rs

```diff
@@ -88,7 +87,7 @@ pub fn tembo_credentials_file_path() -> String {
     tembo_home_dir() + "/credentials"
 }
 
-pub fn list_context() -> Result<Context> {
+pub fn list_context() -> Result<Context, anyhow::Error> {
```

Comment:
`Result<T, E>` Type: In Rust, `Result` is an enum used for error handling. It has two variants: `Ok(T)`, which represents success and contains a value of type T, and `Err(E)`, which represents an error and contains an error of type `E`.

`anyhow::Error`: anyhow is a crate in Rust that provides a convenient way to handle errors. `anyhow::Error` is commonly used when you don't need or want to create specific error types for each kind of error.

Reply:
Nice! Thanks for the explanation.